pcsx2/R5900.cpp

```diff
@@ -540,6 +549,49 @@ void __fastcall eeGameStarting()
 	}
 }
 
+// Count arguments, save their starting locations, and replace the space separators with null terminators so they're separate strings
+int ParseArgumentString(u32 arg_block)
+{
+	if (!arg_block)
+		return 0;
+
+	int argc = 1; // one arg is guaranteed at least
+	g_argPtrs[0] = arg_block; // first arg is right here
+	bool wasSpace = false; // status of last char. scanned
+	int args_len = strlen((char *)PSM(arg_block));
+	for (int i = 0; i < args_len; i++)
+	{
+		char curchar = *(char *)PSM(arg_block + i);
+		bool isSpace = curchar == ' ' ? true : false;
+		bool isNull = curchar == '\0' ? true : false;
```

Comment:
redundant ternary operators, coverity will complain about this.

Reply:
Ah, okay.  I don't know why I did it that way.  I should just change it to "bool isSpace = (curchar == ' '); bool isNull = (curchar == '\0');", right?